client/src/app/shared/components/sorting-list/sorting-list.component.ts

```diff
@@ -27,27 +29,56 @@ import { EmptySelectable } from '../empty-selectable';
 })
 export class SortingListComponent implements OnInit {
     /**
-     * The Input List Values
+     * Sorted and returned
+     */
+    public array: Array<Selectable>;
+
+    /**
+     * Declare the templateRef to coexist between parent in child
+     */
+    @ContentChild(TemplateRef)
+    public templateRef: TemplateRef<Selectable>;
+
+    /**
+     * Determine whether changes to the list occur directly after drag'n'drop
```

Comment:
changes to what: the sortEvent or the listOfSeelctables?

Reply:
What I tried to express was:
if live: sort events should immediately fire events (i.e in list of speakers)
if not: event should be fired after the user hits a save-button (i.e sort motions in categories)